packages/components/date-picker/src/date-picker-com/panel-month-range.vue

```diff
@@ -118,7 +118,7 @@ export default defineComponent({
     const leftDate = ref(dayjs().locale(lang.value))
     const rightDate = ref(dayjs().locale(lang.value).add(1, 'year'))
 
-    const hasShortcuts = computed(() => !!shortcuts.length)
+    const hasShortcuts = computed(() => !!shortcuts.value.length)
```

Comment:
I think we can do better like:

```ts
const hasShortcuts = computed(() => shortcuts.value.length > 0)
```

the negation here is making no sense to me.